### ISSUE TITLE
Modeling (ANN + Other Models)

### ISSUE THREAD

**Issue by SnowyPlums** [SnowyPlums/ML-Streamlit#3]
Design ANN architecture: Decide the number of layers, neurons, activation functions, and optimizer.

Compile and configure ANN model: Set up loss function (categorical_crossentropy) and optimizer (e.g. adam)

Train ANN model

Validate ANN model

Test ANN model

-------------------------------------------------
Other supervised models I also want to use will go in this issue
